src/features/DrugChart/DrugChart.jsx

```diff
@@ -0,0 +1,231 @@
+import React from "react";
+import PropTypes from "prop-types";
+import moment from "moment";
+import Calendar from "../Calendar/Calendar";
+import CalendarHeader from "../CalendarHeader/CalendarHeader";
+import "./DrugChart.scss";
+
+export const TransformDrugChartData = (drugChartData) => {
+  const transformedDrugChartData = [];
+  drugChartData.forEach((schedule) => {
```

Comment:
Coding nitpick. using a forEach to run a process on each item in an array to push a transformed into into another array is the EXACT usecase of a .map() function. Please use map, that is more clear